app/models/concerns/rubygem_searchable.rb

```diff
@@ -69,12 +69,12 @@ def self.elastic_search(q) # rubocop:disable Metrics/MethodLength
         query do
           function_score do
             query do
-              filtered do
+              bool do
                 # Main query, search in name, summary, description
-                query do
+                must do
                   query_string do
                     query q
-                    fields ['name^3', 'summary^1', 'description']
+                    fields ['name^5', 'summary^3', 'description']
```

Comment:
Boost of `^1` is the default. A *keyword* in the summary should be more important than that in the description.